axiom/run.go

```diff
@@ -201,8 +204,12 @@ func (c *axiomClient) do(ctx context.Context, rawURL string, id int, body, v any
 	req.Header.Set("User-Agent", "axiom-clickbench/"+c.version)
 	req.Header.Set("X-Axiom-Org-Id", c.org)
 
+	if c.label == "" {
+		c.label = "clickbench" // default profile label if not set
```

Comment:
I wouldn't mutate c.axiomClient here, since at some point we may change the code to run concurrently and this would then be a data race.

Instead, add I'd add this logic to the constructor in `newAxiomClient`

Reply:
Perfect - thanks